index.html

```diff
@@ -1188,10 +1190,10 @@ <h3>Multikey</h3>
 
             <p>
 In the example above, the `publicKeyMultibase` value starts with the letter `z`,
-which is the [[?MULTIBASE]] header that conveys that the binary data is
-base58-encoded using the Bitcoin base-encoding alphabet. The decoded binary data
-[[?MULTICODEC]] header is `0xed`, which specifies that the remaining data
-is a 32-byte raw Ed25519 <a>public key</a>.
+which is the <a href="#multibase-0">Multibase</a> header that conveys that the
+binary data is base-58-btc-encoded using the Bitcoin base-encoding alphabet. The
```

Comment:
base-58-btc-encoded is not defined in the specification.  Given there's a normative dependency on it, it must be.

Reply:
There is a normative section on Multibase and base-58-btc here:

https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/196.html#multibase-0

There is now a normative section on how to do arbitrary base-encoding here (with normative algorithms and example code on how to implement those algorithms):

https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/196.html#base-encoding

base-58-btc-encoding is now normatively defined in the specification. If you disagree, please provide concrete text that would address your concern.